plenum/test/monitoring/test_revival_spike_resistant_ema_throughput_measurement.py

```diff
@@ -98,7 +98,35 @@ def test_rsr_ema_tm_past_windows_processed_on_get_throughput(tm):
 
     # [30, 45)
     tm.get_throughput(42)
-    assert tm.window_start_ts == 30
+    assert tm.window_start_ts == 15
```

Comment:
Because get_throughput recalculated throughput value early, but now in the safe start we do it only in add_request().
After `tm.add_request(16)` value of `tm.window_start_ts` will 15. And after `tm.get_throughput(42)` it will not be change.